src/bindings/pygaia/scripts/classification/generate_classification_project.py

```diff
@@ -76,19 +112,27 @@ def generateProject(groundtruth_file, filelist_file, project_file, datasets_dir,
 
 
 if __name__ == '__main__':
-    parser = OptionParser(usage = '%prog [options] groundtruth_file filelist_file project_file datasets_dir results_dir')
-
-    options, args = parser.parse_args()
-
-    try:
-        groundtruth_file = args[0]
-        filelist_file = args[1]
-        project_file = args[2]
-        datasets_dir = args[3]
-        results_dir = args[4]
-    except:
-        parser.print_help()
-        sys.exit(1)
-
-    generateProject(groundtruth_file, filelist_file, project_file, datasets_dir, results_dir)
-
+    parser = ArgumentParser()
+
+    parser.add_argument('groundtruth_file')
+    parser.add_argument('filelist_file')
+    parser.add_argument('project_file')
+    parser.add_argument('datasets_dir')
+    parser.add_argument('results_dir')
+    parser.add_argument('-s', '--seed', default=None)
+    parser.add_argument('-c', '--cluster_mode', action='store_true')
```

Comment:
small thing, but typically long arguments use a - instead of a _. I changed this in `train_model`. Did we want to consider changing the name like you suggested? `--fork`?